test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -132,7 +133,7 @@ class ExtProcIntegrationTest : public HttpIntegrationTest,
       }
 
       std::string ext_proc_filter_name = "envoy.filters.http.ext_proc";
-      if (downstream_filter_) {
+      if (config_option.downstream_filter) {
```

Comment:
This change is a minor cleanup of a previous PR: https://github.com/envoyproxy/envoy/pull/33273#discussion_r1563066417